core/src/main/java/com/linecorp/armeria/client/retry/RetryRequestStrategy.java

```diff
@@ -25,14 +27,17 @@
  */
 @FunctionalInterface
 public interface RetryRequestStrategy<I extends Request, O extends Response> {
+
+    CompletableFuture<Boolean> NO_RETRY = CompletableFuture.completedFuture(false);
```

Comment:
I was using it in some other places, but those are all removed during refactoring. I will just remove it as well.